support/scheduler-client/client.go

```diff
@@ -8,15 +8,20 @@ package scheduler
 
 import (
 	"bytes"
+	"encoding/json"
 	"fmt"
+	"github.com/edgexfoundry/edgex-go/core/domain/models"
 	"io"
+	"io/ioutil"
 	"net/http"
-
-	"github.com/edgexfoundry/edgex-go/core/domain/models"
 )
 
 // Common http const
 const (
+	HttpGetMethod      = "GET"
```

Comment:
Use Methods in `net/http` instead of defining it here.
Example: https://github.com/edgexfoundry/edgex-go/blob/2d6ce4d073a7df452ee3479b85b00c1933455776/export/distro/http.go#L39